pkg/controller/controller.go

```diff
@@ -220,12 +238,24 @@ func New(ctx context.Context, cfg *config.Config, kubeclientset kubernetes.Inter
 		wfLauncher = launchplan.NewFailFastLaunchPlanExecutor()
 	}
 
+	namespaceFilterR := func() *regexp.Regexp {
+		if cfg.NamespaceFilter != "" {
+			r, err := regexp.Compile(cfg.NamespaceFilter)
+			if err != nil {
+				logger.Warningf(ctx, "[%s] is not a valid regular expression, skipping namespace filtering. Error : [%v]", cfg.NamespaceFilter, err)
```

Comment:
This should return an error and not try to recover from bad config errors.

Reply:
Yeah that might be better. I wasn't so sure whether this should be treated as failure. This could be done better as suggested by https://github.com/lyft/flytepropeller/pull/90#discussion_r393113304